diff --git a/lib/blockchainexplorers/insight.js b/lib/blockchainexplorers/insight.js
index 059a024..7eea164 100644
--- a/lib/blockchainexplorers/insight.js
+++ b/lib/blockchainexplorers/insight.js
@@ -12,7 +12,7 @@ function Insight(opts) {
   $.checkArgument(_.contains(['livenet', 'testnet'], opts.network));
   $.checkArgument(opts.url);
 
-  this.apiPrefix = opts.apiPrefix || '/api';
+  this.apiPrefix = opts.apiPrefix || '/insight-api';
   this.network = opts.network || 'livenet';
   this.hosts = opts.url;
   this.userAgent = opts.userAgent || 'bws';
